modules/gitops-release-notes-1-3-2.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-3-2_{context}"]
+= Release notes for {gitops-title} 1.3.2
+
+{gitops-title} 1.3.2 is now available on {product-title} 4.6, 4.7, 4.8, and 4.9.
+
+[id="new-features-1-3-2_{context}"]
+== New features
+
+In addition to the fixes and stability improvements, the following sections highlight what is new in {gitops-title} 1.3.2:
+
+* Upgraded Argo CD to version *2.1.8*
+
+* Upgraded Dex to version *2.30.0*
+
+[id="fixed-issues-1-3-2_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Previously, the {gitops-title} did not appear in the `OperatorHub` when you filtered by "Disconnected" under the *Infrastructure Features*, since the Operator did not have the related annotation set in its CSV. With this update, the {gitops-title} is annotated with "Disconnected Cluster" as an *Infrastructure Feature* link:https://issues.redhat.com/browse/GITOPS-1539[GITOPS-1539]
+
+* When using an "Namespace-scoped" argocd, for example, argocd that is not scoped to *All Namepsaces* in a cluster, the {gitops-title} dynamically maintains a list of managed namespaces. These namespaces include the `argocd.argoproj.io/managed-by` label. This list of namespaces is stored in a cache in argocd *Settings > Clusters > "in-cluster" > NAMESPACES*. Before this update, if you deleted one of these namespaces, the Operator ignored that, and the namespace remained in the list. This behavior broke the *CONNECTION STATE* in that cluster configuration, and all sync attempts resulted in errors. For example:
++
+[source,text]
+----
+Argo service account does not have <random verb> on <random resource type> in namespace the-namespace-you-deleted. 
+----
++
+This bug is fixed. link:https://issues.redhat.com/browse/GITOPS-1521[GITOPS-1521]
+
+* {gitops-title} is annotated as "Deep Insights" under the *Capability* Level. link:https://issues.redhat.com/browse/GITOPS-1519[GITOPS-1519]
```

Comment:
This should be The {gitops-title} Operator
*Deep Insights*
Suggest: With this update, the {gitops-title} Operator has been annotated with the *Deep Insights* capability level.